bridges/snowbridge/pallets/system-frontend/src/tests.rs

```diff
@@ -126,3 +128,106 @@ fn test_switch_operating_mode() {
 		assert_ok!(EthereumSystemFrontend::register_token(origin, asset_id, asset_metadata),);
 	});
 }
+
+#[test]
```

Comment:
You're not testing `SwapError` in this file

Reply:
Added a test in https://github.com/paritytech/polkadot-sdk/pull/8271/commits/b8b8b4b3d75f7b40185b4772e742ba09d33f3182.